### ISSUE TITLE
Possible RSS Enhancement

### ISSUE THREAD

**Issue by mastafrooper** [hadouken/hadouken#198]
Currently in the RSS plugin you specify a feed and all its options and filters are linked to that feed.

I was thinking it may work better to separate the feed and it options away from the filtering part.

For example if I have one feed and need multiple filters for it with multiple save locations I would need to specify the same feed numerous times with the different options. It seems this would mean it would check this feed multiple times based on how many time I have it setup.

By separating the feed and options out from the filtering I could just specify the feed and have it only checked once. Then each separate filter could just apply against that one feed.

For example:

```
{
  "feeds":
  [
    {
      "feedID": "feed1",
      "url": "http://some-rss.net/feed",
      "ttl": 2
    }
  ]
}
```

```
{
  "feedfilters":
  [
    {
      "feedID": "feed1",
      "filter": [ "regex", "720p", "NUKED" ]
      "savePath": "C:\somedirectory"
      "label or tags": "somelabel"
    }
    {
      "feedID": "feed1",
      "filter": [ "filter2", "1080p", "PROPER" ]
      "savePath": "C:\someotherdirectory"
      "label or tags": "someotherlabel"
    }
  ]
}
```

Just some thoughts I had on it anyway.

**Comment by vktr** [comment]
This is a good idea, and we had something like this in v4. I'm thinking more in the lines of,

``` js
{
  "feeds":
  [
    {
      "url": "http://some-feed.com",
      "ttl": 2,
      "filters":
      [
        {
          "filter": [ "regex", "720p", "NUKED" ],
          "options":
          {
            "savePath": "C:/Some dir"
          }
        }
      ]
    }
  ]
}
```

This way we keep the filters in the feed configuration, but allow multiple filters per feed.

**Comment by mastafrooper** [comment]
Yeah. That would certainly work. Probably much better than my idea as well. Hence why I don't code.
